td-agent/Rakefile

```diff
@@ -168,6 +168,21 @@ def tar_command
   end
 end
 
+def use_ruby3?
+  return false unless File.exist?("/etc/os-release")
+  os_release_entries = File.open("/etc/os-release").readlines.map(&:strip)
+  os_release_entries.include?("UBUNTU_CODENAME=jammy") and os_release_entries.include?("ID=ubuntu")
+end
```

Comment:
I guess that it is not enough to switch using Ruby 3.1 or Ruby 2.7 on creating tarball and building.
We should bundle Ruby 3.1 and Ruby 2.7 tarballs on the stable source and switch them with this method.

Reply:
You are right, we don't want to depend on host OS for building GNU/Linux version of td-agent.

Reply:
Changed to bundle both of versions.
Thanks.